doc/release-process.md

```diff
@@ -12,10 +12,35 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+We aim to cut a regular release every 3-4 months, approximately twice as frequent as major Bitcoin Core releases. Every second release should be published one month before the feature freeze of the next major Bitcoin Core release, allowing sufficient time to update the library in Core.
+
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+./autogen.sh && ./configure && make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./autogen.sh && ./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
+gcc -o ecdsa examples/ecdsa.c $(PKG_CONFIG_PATH=$dir/lib/pkgconfig pkg-config --cflags --libs libsecp256k1) -Wl,-rpath,"$dir/lib" && ./ecdsa
+```
+3. Check installation with CMake:
+```shell
+dir=$(mktemp -d)
+mkdir build && cmake -B build -DCMAKE_INSTALL_PREFIX=$dir -DBUILD_SHARED_LIBS=OFF && cmake --build build --target install && ls -l $dir/include $dir/lib*
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib*/ -l secp256k1 -Wl,-rpath,"$dir/lib" && ./ecdsa
```

Comment:
Okay, I think `-DBUILD_SHARED_LIBS=OFF` (= static library) together with `-rpath` (= search path for dynamic linker) does not make sense.

I think we should just stick either to the shared library for both build systems, or, if we want some more variety in the sanity checks, use the static library here. (And drop the rpath then.)

Reply:
The current PR uses the shared library for both build systems.

Reply:
> The current PR uses the shared library for both build systems.

Are you sure? I don't think so. These CMake commands here shouldn't build a shared library.

Reply:
Oh sorry, I thought I had removed the `-DBUILD_SHARED_LIBS=OFF` after testing it.

Reply:
Fixed now